### PR TITLE
[hacktogether] Add support for micro-segmentation creation within Athenz UI.

### PR DESCRIPTION
**Pull request by jothi-prasad**

### CODE REVIEW

**Review by abvaidya — CHANGES_REQUESTED** [review]
Please change copyright to Athenz Authors.

**Comment by jothi-prasad** [comment]
> Please change copyright to Athenz Authors.

is there a sample to follow.. i see the same pattern followed in other files too for ex: https://github.com/AthenZ/athenz/blob/master/ui/src/components/policy/AddPolicy.js

File: ui/src/components/microsegmentation/AddSegmentation.js [hunk 1]
```diff
@@ -0,0 +1,783 @@
...
+                );
+            })
+            .catch((err) => {
+                this.setState({
+                    errorMessage: RequestUtils.xhrErrorCheckHelper(err),
+                });
+            });
+    }
+
+    ServiceChange(chosen, key) {
```
Comment (pratikgote) [inline ui/src/components/microsegmentation/AddSegmentation.js:588]: changeService?

File: ui/src/components/microsegmentation/AddStaticInstances.js [hunk 2]
```diff
@@ -0,0 +1,278 @@
...
+import InputLabel from '../denali/InputLabel';
+import Member from '../member/Member';
+import styled from '@emotion/styled';
+import { colors } from '../denali/styles';
+import AddModal from '../modal/AddModal';
+import RequestUtils from '../utils/RequestUtils';
+import InputDropdown from '../denali/InputDropdown';
+import Icon from '../denali/icons/Icon';
+
+const Resourcetypes = [
```
Comment (pratikgote) [inline ui/src/components/microsegmentation/AddStaticInstances.js:27]: can we move this to consts file?

File: ui/src/components/microsegmentation/AddStaticInstances.js [hunk 3]
```diff
@@ -0,0 +1,278 @@
...
+    }
+
+    addMember() {
+        let resourceVal = this.state.resourceValue;
+        let members = this.state.members;
+
+        if (!resourceVal) {
+            return;
+        }
+        let names = (resourceVal || '')
```
Comment (pratikgote) [inline ui/src/components/microsegmentation/AddStaticInstances.js:98]: can we move this code to NameUtils.js and replace it everywhere this code is being used?

**Review by pratikgote — CHANGES_REQUESTED** [review]

**Comment by abvaidya** [comment]
Chandu will address review comments in his PR.

**Pull request merged**